UnitsNet/QuantityValue.cs

```diff
@@ -90,10 +113,235 @@ private QuantityValue(decimal val)
+            {
+                return a._valueDecimal.GetValueOrDefault() < b._valueDecimal.GetValueOrDefault();
+            }
+            else if (a.IsDecimal)
+            {
+                return a._valueDecimal.GetValueOrDefault() < (decimal)b._value.GetValueOrDefault(0); // other._value cannot be null here
+            }
+            else if (b.IsDecimal)
+            {
+                return (decimal)a._value.GetValueOrDefault(0) < b._valueDecimal.GetValueOrDefault();
+            }
+            else
+            {
+                return a._value < b._value;
+            }
+        }
+
+        /// <summary>
+        /// Greater-than-or-equal operator
+        /// </summary>
+        public static bool operator >=(QuantityValue a, QuantityValue b)
+        {
+            if (a.IsDecimal && b.IsDecimal)
+            {
+                return a._valueDecimal.GetValueOrDefault() >= b._valueDecimal.GetValueOrDefault();
+            }
+            else if (a.IsDecimal)
+            {
+                return a._valueDecimal.GetValueOrDefault() >= (decimal)b._value.GetValueOrDefault(0); // other._value cannot be null here
+            }
+            else if (b.IsDecimal)
+            {
+                return (decimal)a._value.GetValueOrDefault(0) >= b._valueDecimal.GetValueOrDefault();
+            }
+            else
+            {
+                return a._value >= b._value;
+            }
+        }
+
+        /// <summary>
+        /// Less-than-or-equal operator
+        /// </summary>
+        public static bool operator <=(QuantityValue a, QuantityValue b)
+        {
+            if (a.IsDecimal && b.IsDecimal)
+            {
+                return a._valueDecimal.GetValueOrDefault() <= b._valueDecimal.GetValueOrDefault();
+            }
+            else if (a.IsDecimal)
+            {
+                return a._valueDecimal.GetValueOrDefault() <= (decimal)b._value.GetValueOrDefault(0); // other._value cannot be null here
+            }
+            else if (b.IsDecimal)
+            {
+                return (decimal)a._value.GetValueOrDefault(0) <= b._valueDecimal.GetValueOrDefault();
+            }
+            else
+            {
+                return a._value <= b._value;
+            }
+        }
+
+        /// <summary>
+        /// Returns the negated value of the operand
+        /// </summary>
+        /// <param name="v">Value to negate</param>
+        /// <returns>-v</returns>
+        public static QuantityValue operator -(QuantityValue v)
+        {
+            if (v._valueDecimal.HasValue)
+            {
+                return new QuantityValue(-v._valueDecimal.Value);
+            }
+            else
+            {
+                return new QuantityValue(-v._value.GetValueOrDefault());
+            }
+        }
+
+        /// <summary>
+        /// Multiplication operator. Note that this performs an operation on a raw value, as no unit is associate the the value at this point.
```

Comment:
I initially would argue that it is redundant to explain the unit is not involved, but I realize the name `QuantityValue` is misleading. It was originally only used internally.

It might be a good idea to rename it to `Number` or something more generic in v5, to make it more clear it does not really have anything to do with quantities on its own. This would be a seaparate PR anyhow.

A minor suggestion:
```xml
<remarks>
This performs an operation on the numeric value of a quantity, there is no unit or conversions involved.
</remarks>
```

Reply:
Ok, sounds like a plan. I added this to make it obvious that `quantityA * quantityB` is something very different from `quantityA.QuantityValue * quantityB.QuantityValue`.